config.py

```diff
@@ -433,7 +466,8 @@
 # ~~~~~~~
 # >>> stim_channel = 'STI 014'  # or 'STI101'
 
-stim_channel = 'STI 014'
+# XXX not needed if bids events are present
+# stim_channel = 'STI 014'
```

Comment:
also you should know this from _channels.tsv if you really need stim channel for some reason